e2e/ansible/roles/k8s-openebs-operator/tasks/main.yml

```diff
@@ -38,19 +38,42 @@
   shell: source ~/.profile; kubectl apply -f {{ openebs_operator_alias }} 
   args:
     executable: /bin/bash
+  register: op_out
 
-- name: Confirm maya-apiserver pod is running
-  shell: source ~/.profile; kubectl get pods
+- name: Place stdout into temp file
+  shell: echo {{item}} >> {{ ansible_env.HOME }}/op_out.log
+  args:
+    executable: /bin/bash
+  with_items: "{{op_out.stdout_lines}}"
+
+- name: Get deployment label names 
+  shell: cat {{ ansible_env.HOME }}/op_out.log | grep deployment | awk '{print $2}'
```

Comment:
This can be done while running kubectl apply command above.

``` 
shell: source ~/.profile; kubectl apply -f {{ openebs_operator_alias }} | grep deployment | awk '{print $2}'
```

Writing it out into a file and applying grep on that file may result in an extra step.

Reply:
Agreed, and done.